schema/filter_artworks.js

```diff
@@ -60,21 +63,39 @@ export const FilterArtworksCounts = {
 
 export const FilterArtworksType = new GraphQLObjectType({
   name: "FilterArtworks",
+  interfaces: [NodeInterface],
+  isTypeOf: obj => has(obj, "hits") && has(obj, "aggregations"),
   fields: () => ({
+    __id: {
+      type: new GraphQLNonNull(GraphQLID),
+      description: "The ID of the object.",
+      resolve: ({ options }) => {
+        return toGlobalId("FilterArtworks", JSON.stringify(options))
+      },
+    },
     aggregations: ArtworkFilterAggregations,
     artworks_connection: {
       type: artworkConnection,
       deprecationReason: "Favour artwork connections that take filter arguments.",
-      args: pageable(),
-      resolve: ({ hits, aggregations }, options) => {
-        if (!aggregations || !aggregations.total) {
-          throw new Error("This query must contain the total aggregation")
-        }
-        const relayOptions = parseRelayOptions(options)
-        return connectionFromArraySlice(hits, options, {
-          arrayLength: aggregations.total.value,
-          sliceStart: relayOptions.offset,
-        })
+      args: pageable({
+        sort: {
+          type: GraphQLString,
+        },
+      }),
+      resolve: ({ options: gravityOptions }, args, request, { rootValue: { accessToken } }) => {
+        const relayOptions = parseRelayOptions(args)
+        return gravity
```

Comment:
no data loader, but I can live with this

Reply:
One thing per PR remember! That can be a different PR that _just_ updates data loaders :)